packages/docs/docs/api-reference/LayoutGridField.md

```diff
@@ -0,0 +1,267 @@
+
+```json
+{
+  "ui:col": [
+    { "name": "innerField", "fullWidth": true },
+    { "name": "inner.grandChild", "convertOther": true },
+    ...
+  ]
+}
+```
+
+More complicated `ui:col` definition, without additional `GridTemplate` props and form 2 children, one being a
+one-off `render` functional component without a non-field `name` identifier
+
+```json
+{
+  "ui:col": [
+    "innerField",
+    {
+      "render": "WizardNavButton",
+      "isNext": true,
+      "size": "large"
+    }
+  ]
+}
+```
+
+Most complicated `ui:col` definition, additional `GridTemplate` props and form 1, 2 and 3 children (this example
+uses @mui/material/Grid2 native props):
+
+```json
+{
+  "ui:col": {
+    "size": { "md": 4 },
+    "className": "GridColumn",
+    "children": [
+      "innerField",
+      { "name": "inner.grandChild", "convertOther": true },
+      { "name": "customRender", "render": "CustomRender", "toSpread": "prop-value" },
+      { "ui:row|ui:condition": ... },
+      ...
+    ]
+  }
+}
+```
+
+> NOTE: If a `name` prop does not exist or its value does not match any field in a schema, then it is assumed to be a
+> custom `render` component. If the `render` prop does not exist, a null render will occur. If `render` is a
+> string, its value will be looked up in the `formContext.lookupMap` first before defaulting to a null render. If the
+> resulting looked up `render` is not a function (React components are functions), then it will be a null render.
+
+## ui:columns
+
+The `ui:columns` grid element is syntactic sugar to specify a set of `ui:col` columns that all share the same set of
+native `GridTemplate` props. In other words rather than writing the following configuration that renders a
+`<GridTemplate>` element with 3 `<GridTemplate column className="GridColumn col-md-4">` nodes and 2
+`<GridTemplate column className="col-md-6">` nodes within it (one for each of the fields contained in the `children`
+list):
+
+```json
+{
+  "ui:row": {
+    "children": [
+      {
+        "ui:col": {
+          "className": "GridColumn col-md-4",
+          "children": ["innerField"]
+        }
+      },
+      {
+        "ui:col": {
+          "className": "GridColumn col-md-4",
+          "children": ["inner.grandChild"]
+        }
+      },
+      {
+        "ui:col": {
+          "className": "GridColumn col-md-4",
+          "children": [{ "name": "inner.grandChild2" }]
+        }
+      },
+      {
+        "ui:col": {
+          "className": "col-md-6",
+          "children": ["innerField2"]
+        }
+      },
+      {
+        "ui:col": {
+          "className": "col-md-6",
+          "children": ["inner.grandChild3"]
+        }
+      }
+    ]
+  }
+}
+```
+
+One can write this instead:
+
+```json
+{
+  "ui:row": {
+    "children": [
+      {
+        "ui:columns": {
+          "className": "GridColumn col-md-4",
+          "children": ["innerField", "inner.grandChild", { "name": "inner.grandChild2", "convertOther": true }]
```

Comment:
what is this `convertOther` property? I thought it was some custom/arbitrary property but I don't see it in the previous example?

Reply:
Ah, its a custom prop that we have in our world. I'll switch it to something like `"fullWidth": true`